core/src/main/java/com/linecorp/armeria/common/stream/AbstractStreamMessageDuplicator.java

```diff
@@ -62,7 +64,7 @@
  * @param <T> the type of elements
  * @param <U> the type of the publisher and duplicated stream messages
  */
-public abstract class AbstractStreamMessageDuplicator<T, U extends StreamMessage<? extends T>>
+public abstract class AbstractStreamMessageDuplicator<T, U extends StreamMessage<T>>
```

Comment:
Well, I'm not sure, but we don't need it. We can, also, remove the following `SafeCloseable` because we don't use this with `try-with-resources` anymore...